kube/manifest/do-migration/migration-seed-aws-pod.yaml

```
@@ -0,0 +1,149 @@
+    
+
+    echo "Verify service status..."
+    echo "=== SSH service status ==="
+    ps aux | grep sshd | grep -v grep || echo "SSH service not running"
+    
+    echo "=== rsync daemon status ==="
+    ps aux | grep rsync | grep -v grep || echo "rsync daemon not running"
+    
+    echo "=== port listening status ==="
+    netstat -tlnp | grep -E ':(22|873)' || echo "port not listening"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)